lib/platform-stats.js

```diff
@@ -11,38 +11,32 @@ const debug = createDebug('spark:platform-stats')
  * @param {import('./preprocess.js').Measurement[]} allMeasurements
  */
 export const updatePlatformStats = async (pgClient, honestMeasurements, allMeasurements) => {
-  const participants = new Set(honestMeasurements.map(m => m.participantAddress))
-  await updateDailyParticipants(pgClient, participants)
+  const participantsMap = await mapParticipantsToIds(pgClient, new Set(allMeasurements.map(m => m.participantAddress)))
+  await updateDailyParticipants(pgClient, Array.from(participantsMap.values()))
   await updateDailyStationStats(pgClient, honestMeasurements, allMeasurements)
+  await updateStationsAndParticipants(pgClient, honestMeasurements, allMeasurements, participantsMap)
 }
 
 /**
  * @param {import('pg').Client} pgClient
  * @param {import('./preprocess.js').Measurement[]} honestMeasurements
  * @param {import('./preprocess.js').Measurement[]} allMeasurements
- * @param {Date} day
+ * @param {object} options
```

Comment:
This argument is also optional

```suggestion
 * @param {object} [options]
```